### PR TITLE
feat: initial setup and added pages and routes

### PR DESCRIPTION
**Pull request by deekshithmd**
Added pages and routes for note taking app
Files to check
Homepage
Sidebar
Notes
Archives
Label
Trash

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Everything is good to go.
You can use utility classes for CSS and variable for the repeating units and stuffs.

File: src/components/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./navigation.css";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+export const Navigation = () => {
+  const navigate = useNavigate();
```
Comment (sanjitsarkar) [inline src/components/Navigation/Navigation.jsx:6]: **Suggestion:** Remove unused variable

File: src/components/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Calistoga&display=swap');
+*{
+    font-family: 'Calistoga', cursive;
+}
+.brand{
+    line-height: 13vh;
+    text-decoration: none;
```
Comment (sanjitsarkar) [inline src/components/Navigation/navigation.css:7]: **Suggestion:** Don't put large values for line-height.

File: src/pages/Authentication/authentication.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+.form-data{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    width: fit-content;
+    height: fit-content;
+    background-color: var(--background-color);
+    padding: 2rem;
+    /* border-radius: 10px;
```
Comment (sanjitsarkar) [inline src/pages/Authentication/authentication.css]: **Suggestion:** Remove commented code.

**Review by sanjitsarkar — APPROVED** [review]

File: src/components/Navigation/Navigation.jsx (hunk 1 above)
Comment (deekshithmd) [reply src/components/Navigation/Navigation.jsx:6]: Ok, I will check that

File: src/components/Navigation/navigation.css (hunk 2 above)
Comment (deekshithmd) [reply src/components/Navigation/navigation.css:7]: ok, I will look into it

File: src/pages/Authentication/authentication.css (hunk 3 above)
Comment (deekshithmd) [reply src/pages/Authentication/authentication.css]: I will check that

**Comment by deekshithmd** [comment]
> Everything is good to go. You can use utility classes for CSS and variable for the repeating units and stuffs.

Need to look into this concept, thank you for suggesting

**Review by saurabhchirde — APPROVED** [review]
Clean and easy to read code. 
You can proceed to merge.

**Pull request merged**